### ISSUE TITLE
Support for python 3.11

### ISSUE THREAD

**Issue by detrin** [analyticalmindsltd/smote_variants#78]
Do you plan on adding support for python3.11?

**Comment by gykovacs** [comment]
Yes, it has been just added. I couldn't create the 0.7.2 release yet as one of the dependencies (metric-learn) struggles with supporting the latest numpy, but it should work by cloning from this repo. As soon as metric-learn is fixed (as far as I know it already is, they just did not release to PyPI yet), I will also release 0.7.2 to PyPI.
